docs/run/index.md

```diff
@@ -8,7 +8,19 @@ parent:
 
 # Start a Blockchain
 
-One of your first actions is to start your blockchain. To start your blockchain, use the `starport serve` command.
+Blockchains are decentralized applications that in production run (often) the same software on many validator nodes by different people/entities. To launch a blockchain in production entities coordinate on the launch process together and start their nodes.
+
+During development, however, a blockchain can be started locally on a single validator node. This is very convenient, because you can quickly restart a chain and iterate faster. Starting a chain on single node in development is similar to starting a traditional web application on a local server.
+
+## Start a Blockchain Node in Development
+
+Switch to a directory that contains a blockchain scaffolded with Starport and run the following command:
+
+```
+starport serve
+```
+
+This command initializes a chain, builds the code, starts a single validator node and starts watching for file changes. Whenever a file is changed, Starport will reinitialise the chain, rebuild and start the chain again. Starport tries to preserve the state of the chain, if the changes to source code are compatible with the previous state. This is very useful for development purposes. `starport serve` command is a development tool, it should not be used in production environment. See below how to run a chain in production.
```

Comment:
```suggestion
This command initializes a chain, builds the code, starts a single validator node, and starts watching for file changes. 

Whenever a file is changed, the chain is automatically reinitialized, rebuilt, and started again. The chain's state is preserved if the changes to the source code are compatible with the previous state. This state preservation is beneficial for development purposes. 

Because the `starport serve` command is a development tool, it should not be used in a production environment. Read on to learn the process of running a blockchain in production. 
```

Reply:
I know it's tempting, but let's try not to make Starport an "actor" here. 

Does this rewrite work to prevent anthropomorphism? I like this change for the active voice and clear actions. wdyt?

Reply:
Sounds good!